UNRELEASED_CHANGELOG.md

```diff
@@ -27,6 +27,15 @@
 - Fixed sending messages using `ChatClient::sendMessage` without explicitly specifying the sender user id.
 
 ### ⬆️ Improved
+- It is now possible to add a interceptor for API calls of Stream API. You can add it in the Build of ChatClient:
```

Comment:
This PR will further complicate this one https://github.com/GetStream/stream-chat-android/pull/2362, let's make sure we figure out how to make the two work together

Reply:
🤔 Ok. Let's merge the #2362 and I'll adapt this PR to it.

Reply:
That PR is now merged. However, I'm not sure how we can make this work with arbitrary interceptors added by our clients. If they ever read the body of the request, they'll break our upload progress reporting code.

See https://github.com/GetStream/stream-chat-android/pull/2362#pullrequestreview-758671037